src/systems/ec/implementations/react/components/site-header-core/src/SiteHeaderCore.jsx

```diff
@@ -57,6 +57,7 @@ const SiteHeaderCore = ({
                 className="ecl-link ecl-link--standalone ecl-site-header-core__language-selector"
                 href={languageSelector.href}
                 data-ecl-language-selector
+                aria-controls="language-list-overlay"
```

Comment:
2 things:
- add the aria attributes only if `languageSelector && languageSelector.overlay` is true
- provide a default value for `aria-epxanded` (`aria-epxanded="false"`)

To go further: add [`aria-modal="true"`](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html) on `LanguageListOverlay` when the modal is open. When it's not open, remove the attribute ([better than `aria-modal="false"`](https://developer.paciellogroup.com/blog/2018/06/the-current-state-of-modal-dialog-accessibility/))